### PR TITLE
fix: Improve the post CTA from a space - MEED-2070  - meeds-io/meeds#54

### PR DESCRIPTION
**Pull request by MayTekayaa**
This change will improve the post CTA from a space.

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue [hunk 1]
```diff
@@ -23,23 +23,46 @@
...
-        class="ms-auto my-auto">
-        <activity-stream-filter />
-      </div>
-    </v-flex>
+    <v-row class="d-flex flex-nowrap" no-gutters>
+      <v-col 
+        cols="9" 
+        lg="8" 
+        md="6" 
+        xl="9">
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue]: Can you please use a simple `div` with classes `d-flex` & `flex-grow-1` & `flex-grow-0` &  & `flex-shrink-1`
 &  & `flex-shrink-0` for a better UX in all screens.

```suggestion
```

Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue [hunk 2]
```diff
@@ -89,9 +107,15 @@ export default {
       return this.canFilter;
     },
     displayToolbar() {
-      return this.userCanPost || this.streamFilterEnabled;
+      return (this.userCanPost || this.streamFilterEnabled) &&this.user;
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue]:
```suggestion
      return (this.userCanPost || this.streamFilterEnabled) && this.user;
```

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue [hunk 3]
```diff
@@ -89,9 +107,15 @@ export default {
...
     },
   },
+  created() {
+    if (!this.user) {
+      this.$userService.getUser(eXo.env.portal.profileOwner)
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue]:
```suggestion
      this.$userService.getUser(eXo.env.portal.userName)
```

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue (hunk 2 above)
Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue (hunk 3 above)
Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue]: done!

**Review by boubaker — APPROVED** [review]

**Pull request merged**